### PR TITLE
Add Greek templates

### PR DESCRIPTION
**Pull request by MIP2000**

### CODE REVIEW

**Comment by MIP2000** [comment]
Thank you for your message. Since i am a native Greek speaker i translated the files word by word. Since there are legal consequences, i kept a very formal tone in both syntax and grammar. I can try again to change the files if you want (use diffirent words that have the same meaning, use a more informal tone in some sentences, etc.) according to your specifications. In this point i would like to clarify that i strictly followed the original files and that every sentence i translated have the exact same meaning as the original one. Please feel free to point me the changes that you want me to add. Thanks again.

File: templates/el/access-default.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Προς κάθε ενδιαφερόμενο:
+
+Ζητώ την πρόσβαση σύμφωνα με το άρθρο 15 του GDPR. Παρακαλείσθε να επιβεβαιώσετε εάν επεξεργάζεστε ή όχι προσωπικά δεδομένα (όπως ορίζονται στο άρθρο 4 παράγραφοι 1 και 2 του GDPR) σχετικά με εμένα.
+
+Σε περίπτωση που επεξεργάζεστε, σας ζητώ πρόσβαση στις ακόλουθες πληροφορίες σύμφωνα με το άρθρο 15 του GDPR:
+1. <italic>all</italic> προσωπικά δεδομένα που με αφορούν που έχετε αποθηκεύσει;
```
Comment (baltpeter) [inline templates/el/access-default.txt]: The 'all' also needs to be translated.

File: templates/el/access-tracking.txt [hunk 2]
```diff
@@ -0,0 +1,28 @@
+Προς κάθε ενδιαφερόμενο:
+
+Ζητώ την πρόσβαση σύμφωνα με το άρθρο 15 του GDPR. Παρακαλείσθε να επιβεβαιώσετε εάν επεξεργάζεστε ή όχι προσωπικά δεδομένα (όπως ορίζονται στο άρθρο 4 παράγραφοι 1 και 2 του GDPR) σχετικά με εμένα.
+
+Σε περίπτωση που επεξεργάζεστε, σας ζητώ να έχω πρόσβαση στις ακόλουθες πληροφορίες σύμφωνα με το άρθρο 15 του GDPR:
+1. <italic>all</italic> προσωπικά δεδομένα που με αφορούν που έχετε αποθηκεύσει;
```
Comment (baltpeter) [inline templates/el/access-tracking.txt]: Same here.

**Comment by baltpeter** [comment]
Thank you.

**Pull request merged**